language-support/ts/codegen/tests/src/DA/Test/Daml2TsUtils.hs

```diff
@@ -0,0 +1,21 @@
+-- Copyright (c) 2020 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+{-# LANGUAGE DerivingStrategies #-}
```

Comment:
Not needed anymore.
```suggestion
```

Reply:
Heh. I must look into hlint with an eye to understanding why that didn't get flagged! :)